### PR TITLE
Equalarea

### PR DESCRIPTION
**Pull request by Bysmutheye**
Hi, this is my recursive equal area layout. I just changed the return to a table so it would work directly. I didn't add any code for importing the icon, nor did i change the naming scheme to fit with the others in this project. I might do that another day but no promises.

### CODE REVIEW

**Comment by JavaCafe01** [comment]
@Bysmutheye Thanks for the pull request! I remember seeing your post on r/unixporn and how cool it was.

Just one more thing before I finish my review, do you have some sort of icon for the layout, or can you make one? You can look under [`icons/layouts`](https://github.com/Nooo37/bling/tree/master/icons/layouts) for some examples.

**Comment by JavaCafe01** [comment]
Also, if you would like to add a preview to the docs, please go ahead

**Comment by Bysmutheye** [comment]
Ok, i added the icon and doc. Should be ready for merge now

**Comment by branwright1** [comment]
Hello, first sorry for dumb question but I'm wondering if layout logo has equal pixels

File: docs/layouts/layout.md [hunk 1]
```diff
@@ -42,3 +43,8 @@ theme.mstab_tabbar_style = "default"        -- style of the tabbar ("default", "
 ![](https://media.discordapp.net/attachments/769673106842845194/780095998239834142/unknown.png)
 
 *screenshot by [branwright](https://github.com/branwright1)*
+
+#### Equal area
+![](https://imgur.com/a/qt3qxyT)
```
Comment (JavaCafe01) [inline docs/layouts/layout.md:48]: When doing imgur links dont forget the `.png` extension

**Review by JavaCafe01 — APPROVED** [review]

File: docs/layouts/layout.md (hunk 1 above)
Comment (JavaCafe01) [reply docs/layouts/layout.md:48]: I can fix this myself, not to worry

**Pull request merged**